assets/wire/actions.js

```diff
@@ -212,6 +213,9 @@ export function fetchItems() {
                 const state = getState();
                 updateRouteParams({
                     q: state.query,
+                    filter: get(state, 'search.activeFilter'),
```

Comment:
Different browser have different limits on saving data to history. `updateRouteParams` fails on firefox, ie, etc after hitting the limit. 
You would see data changing as dispatch for updating the store is before `updateRouteParams` but the URL stops changing.

Reply:
Hi @mdhaman I think the problem is the sixe of the state being too big for firefox to handle so as you say it was hitting the limit. I made the change now to trim the `items` and `itemsById` which were the biggest chunk in state. Now we fetch once the hidtory is popped back.